seisbench/data/geofon.py

```diff
@@ -23,8 +24,65 @@ class GEOFON(BenchmarkDataset):
     GEOFON dataset consisting of both regional and teleseismic picks. Mostly contains P arrivals,
     but a few S arrivals are annotated as well. Contains data from 2010-2013. The dataset will be
     downloaded from the SeisBench repository on first usage.
+
+    The GEOFON dataset is organized in folders, each containing data for one
+    event. The name of each event folder is the event ID.
+
+    GEOFON event ID's are formed using as prefix "gfz" followed by the year as
+    four digits and by a four-character string. For instance, the GEOFON event
+    ID of the 2012 Mw 8.6 Wharton basin earthquake is 'gfz2012hdex'.
+
+    Parametric data (of which we only need the picks) are provided as SeisComP
+    XML. The name of the XML file is the event ID plus '-preferred-only.xml'.
+    This naming is because originally there is also an XML file containing the
+    full processing history, but we don't need that one.
+
+    The waveforms are provided as plain MiniSEED files of approximately 12
+    minutes length, ranging in time from 6 minutes before until 6 minutes after
+    the P onset. There is one MiniSEED file for each combination of network,
+    station, location and channel code. Note that there may not be waveform
+    data for every pick, as some of the waveforms are restricted and we can
+    only make open data available.
+
+    To sum up, for event 'gfz2012hdex' the data directory looks like
+
+    gfz2012hdex/
+        gfz2012hdex-preferred-only.xml
+        AD.DLV..BHE.mseed
+        AD.DLV..BHN.mseed
+        AD.DLV..BHZ.mseed
+        AD.SIM..BHE.mseed
+        AD.SIM..BHN.mseed
+        AD.SIM..BHZ.mseed
+        AD.SZP..BHE.mseed
+        AD.SZP..BHN.mseed
+        AD.SZP..BHZ.mseed
+        AF.CER..BHE.mseed
+        AF.CER..BHN.mseed
+        AF.CER..BHZ.mseed
+        ...
+        ... many more mseed files ...
+        ...
+        WM.MELI..BHE.mseed
+        WM.MELI..BHN.mseed
+        WM.MELI..BHZ.mseed
+        WM.UCM..BHE.mseed
+        WM.UCM..BHN.mseed
+        WM.UCM..BHZ.mseed
+
+    In order to use the GEOFON dataset, in addition to the raw data as
+    described above, there needs to be an inventory file (as FDSN Station XML)
+    for all stream-time combinations. Data files for which no corresponding
+    inventory entries are found will be ignored. The inventory doesn't need to
+    contain instrument responses, as the latter are not needed. The inventory
+    file is currently expected as file with name
+    'inventory_without_response.xml'.
     """
 
+    start_train = "2010-01-01"
```

Comment:
Maybe we should also add an `end_data` field or similar to indicate data until which data was used in the conversion.